soweego/linker/classify.py

```diff
@@ -39,7 +40,7 @@ def cli(target, target_type, model, upload, sandbox, threshold, dir_io):
         if upload:
             _upload(chunk, target, sandbox)
         chunk.to_csv(os.path.join(dir_io, constants.LINKER_RESULT %
-                                  (target, target_type, model)), mode='a', header=True)
+                                  (target, target_type, 'nb')), mode='a', header=True)
```

Comment:
shouldn't the `'nb'` actually by pulled from the `model` argument?

```suggestion
                                  (target, target_type, model)), mode='a', header=True)
```

Reply:
This was a quick temporary solution, because `model` is a path and will cause a failure.
A known issue that will fix this is #242 .